<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
	border: 2px solid #7409c7;
	border-radius: 10px;
	font-size: 36px;
	line-height: 60px;
}

QPushButton::pressed {
	color: black;
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #ffffff, stop: 1 #5462c3);
}

MainWindow {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #5462c3, stop: 1 #ba872c);
}

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="result_show">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>5</y>
      <width>561</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 36pt &quot;PT Sans&quot;;
color: rgb(10, 10, 10);
background-color: rgb(250, 250, 250);
padding: 15px;
border: 2px solid #7409c7;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="number_button_4">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>275</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="number_button_7">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>215</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="number_button_1">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>335</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="operation_button_clear">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>155</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="operation_button_remove">
    <property name="geometry">
     <rect>
      <x>89</x>
      <y>155</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>&lt;-x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="number_button_8">
    <property name="geometry">
     <rect>
      <x>89</x>
      <y>215</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="number_button_2">
    <property name="geometry">
     <rect>
      <x>89</x>
      <y>335</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="number_button_5">
    <property name="geometry">
     <rect>
      <x>89</x>
      <y>275</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="number_button_0">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>395</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="operation_button_dot">
    <property name="geometry">
     <rect>
      <x>169</x>
      <y>395</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="number_button_9">
    <property name="geometry">
     <rect>
      <x>169</x>
      <y>215</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="number_button_3">
    <property name="geometry">
     <rect>
      <x>169</x>
      <y>335</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="number_button_6">
    <property name="geometry">
     <rect>
      <x>169</x>
      <y>275</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="operation_button_mult">
    <property name="geometry">
     <rect>
      <x>249</x>
      <y>275</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="operation_button_sum">
    <property name="geometry">
     <rect>
      <x>249</x>
      <y>395</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="operation_button_sub">
    <property name="geometry">
     <rect>
      <x>249</x>
      <y>335</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="operation_button_div">
    <property name="geometry">
     <rect>
      <x>249</x>
      <y>215</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="operation_button_sin">
    <property name="geometry">
     <rect>
      <x>409</x>
      <y>155</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="operation_button_log">
    <property name="geometry">
     <rect>
      <x>489</x>
      <y>395</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="operation_button_ln">
    <property name="geometry">
     <rect>
      <x>409</x>
      <y>395</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>ln</string>
    </property>
   </widget>
   <widget class="QPushButton" name="operation_button_cos">
    <property name="geometry">
     <rect>
      <x>409</x>
      <y>215</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="operation_button_tan">
    <property name="geometry">
     <rect>
      <x>409</x>
      <y>275</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="operation_button_left_bracket">
    <property name="geometry">
     <rect>
      <x>169</x>
      <y>155</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="operation_button_right_bracket">
    <property name="geometry">
     <rect>
      <x>249</x>
      <y>155</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="operation_button_atan">
    <property name="geometry">
     <rect>
      <x>489</x>
      <y>275</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>atan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="operation_button_sqrt">
    <property name="geometry">
     <rect>
      <x>409</x>
      <y>335</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>sqrt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="operation_button_pow">
    <property name="geometry">
     <rect>
      <x>329</x>
      <y>155</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="operation_button_mod">
    <property name="geometry">
     <rect>
      <x>489</x>
      <y>335</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>mod</string>
    </property>
   </widget>
   <widget class="QPushButton" name="operation_button_asin">
    <property name="geometry">
     <rect>
      <x>489</x>
      <y>155</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>asin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="operation_button_acos">
    <property name="geometry">
     <rect>
      <x>489</x>
      <y>215</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>acos</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_default">
    <property name="geometry">
     <rect>
      <x>59</x>
      <y>465</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">padding-left: 5px;
font-size: 16px;</string>
    </property>
    <property name="text">
     <string>default</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_credit">
    <property name="geometry">
     <rect>
      <x>279</x>
      <y>465</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">padding-left: 5px;
font-size: 16px;</string>
    </property>
    <property name="text">
     <string>credit</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_deposit">
    <property name="geometry">
     <rect>
      <x>379</x>
      <y>465</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">padding-left: 5px;
font-size: 16px;</string>
    </property>
    <property name="text">
     <string>deposit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="operation_button_result">
    <property name="geometry">
     <rect>
      <x>329</x>
      <y>215</y>
      <width>81</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_graph">
    <property name="geometry">
     <rect>
      <x>169</x>
      <y>465</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">padding-left: 5px;
font-size: 16px;</string>
    </property>
    <property name="text">
     <string>graph</string>
    </property>
   </widget>
   <widget class="QPushButton" name="operation_button_x">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>95</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>x</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>96</x>
      <y>100</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="x_show">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QDoubleSpinBox {
background-color: rgb(250,250,250);
border-radius: 10px;
font-size: 24px;
color: black;
padding-left: 10px;
border: 2px solid #7409c7;
}
QDoubleSpinBox::up-button {
	width: 25px;
	color: white;
}
QDoubleSpinBox::down-button {
	width: 25px;
	color: white;
}</string>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::PlusMinus</enum>
    </property>
    <property name="decimals">
     <number>7</number>
    </property>
    <property name="minimum">
     <double>-82222222222222227285213184.000000000000000</double>
    </property>
    <property name="maximum">
     <double>72222222222222223031332868003463168.000000000000000</double>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
